modules/ROOT/pages/ts-deploy-not-in-effect.adoc

```diff
@@ -0,0 +1,38 @@
+= I recently updated and deployed my message flow, however, the configurations haven’t taken effect on the runtime B2B apps. Did I do this incorrectly?
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+As you continue to use Anypoint Partner Manager, you may find the need to update the configuration associated with a previously deployed message flow.
+After updating the configuration, you still find your message flow executing with the previous or older set of configurations.
+
+== Causes
+
+Any of the following can lead to this type of error:
+* You may have not deployed the message flow after making changes to the message flow’s configuration.
+* You may be within the five minute synchronization window for changes from Anypoint Partner Manager to reflect on your B2B integration applications.
+
+
+== Solutions
+
+=== Deploy the Changed Message Flow
+
+Once you make changes to the message flow configuration, you need to deploy it in order for the new configurations to become effective on your B2B integration applications.
+
+To deploy your message flow after making changes:
+
+. From within your Anypoint Partner Manager environment (either Sandbox or Production) select your <Hostname> profile.
+. Select *Message Flows*.
+. Search for your message flow by name.
+. Once you locate the message flow, click on the message flow card to open it up.
+. Confirm that each sections in the interface shows a green checkmark next to it in the flow, and click Deploy to deploy the configurations to Mule.
+. Provide the necessary reason to trigger a new deployment, and wait for the process to complete.
+
+When the deployment is successfully completed, Anypoint partner manager responds with a successful notification.
+
+=== Wait for the Synchronization Window to Complete
```

Comment:
```suggestion
=== Wait for the Synchronization Window to Finish
```
Except consider rewording, because "a window" does not "finish."
(Note that "complete" requires an object; look it up in the style guide.)

Reply:
f